tests/features/list_players.feature

```diff
@@ -0,0 +1,14 @@
+Feature: List Players
+
+    Background: Login into account
+        Given An account with name "744774", email "jaspion@bol.com.br" and password "Senha123" exists
+        And Client is logged into account of name "744774" with password "Senha123"
+
+    Scenario: Account with no players created
+        When List players created in the account
```

Comment:
man entendi nada o nome desse when aqui

Reply:
ah entendi agora

Reply:
man bota que nem os outros tipo Client tries to list all players in their account